opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/OpenTelemetryTraceContextFormatImpl.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opencensusshim;
+
+import static io.opentelemetry.opencensusshim.SpanConverter.mapSpanContext;
+
+import io.opencensus.trace.SpanContext;
+import io.opencensus.trace.propagation.TextFormat;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.propagation.TextMapPropagator;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nullable;
+
+public class OpenTelemetryTraceContextFormatImpl extends TextFormat {
```

Comment:
Per my above comment, you should be able to have just one class for both b3 and w3c